### ISSUE TITLE
Lots of ios-*-outline icons are removed on newest React Native Vector Icons

### ISSUE THREAD

**Issue by kigawas** [GeekyAnts/NativeBase#2351]
## I have gone through these following points
- [x]   Check latest documentation: https://docs.nativebase.io/
- [x]   Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- [x]   Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- [x]   Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- [x]   For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- [x]   For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

### node, npm, react-native, react and native-base version, expo version if used
```
  React Native Environment Info:
    System:
      OS: macOS 10.14
      CPU: x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
      Memory: 296.78 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.11.0 - /usr/local/bin/node
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.0, macOS 10.14, tvOS 12.0, watchOS 5.0
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5014246
      Xcode: 10.0/10A255 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.5.0 => 16.5.0
      react-native: 0.57.1 => 0.57.1
    npmGlobalPackages:
      create-react-native-app: 1.0.0
      react-native-cli: 2.0.1
```

### Expected behaviour

Showing correct icons.

### Actual behaviour

Because lots of `ios-*-outline` icons are removed, it shows question mark now.

### Steps to reproduce

1. `npm install native-base`
2. `npm install react-native-vector-icons`
3. use `home` or `calendar` icon

![image](https://user-images.githubusercontent.com/4182346/46799001-8aa63a80-cd8e-11e8-966a-5d5df61cfc83.png)

### Is the bug present in both iOS and Android or in any one of them?

Only iOS

### Any other additional info which would help us debug the issue quicker.

When I was checking the latest doc of RN vector icons, it only contains these `ios-*-outline` icons:

![image](https://user-images.githubusercontent.com/4182346/46798921-4adf5300-cd8e-11e8-87a1-f611af51484d.png)

Lots of icons [here](https://github.com/GeekyAnts/NativeBase/blob/master/src/basic/Icon/NBIcons.json) were removed.

So it's better to update the json file and update RN vector icons to the latest version

**Comment by SupriyaKalghatgi** [comment]
Fixed with commit https://github.com/GeekyAnts/NativeBase/commit/f841390a7b4c9c88f9c8000689599783d5cfcf09 and https://github.com/GeekyAnts/NativeBase/commit/e678edbf19765421af50a1cdfc8ecd923976656b

Also available on `master`

**Comment by kigawas** [comment]
@SupriyaKalghatgi 
It seems that [this file](https://github.com/GeekyAnts/NativeBase/blob/master/src/basic/Icon/NBIcons.json) was not updated. The `ios-*-outline` icons should be reviewed, I think.

**Comment by alexkyse1** [comment]
I agree with  @kigawas , there is some work left on this issue...

**Comment by OtacilioN** [comment]
Hey folks,

You are right @kigawas I got this bug reported on #2461 and fixed on #2462 by reviewing this file that @alexkyse1 suggested!

Can you take a look  👀 in this PR @SupriyaKalghatgi ?

Thank you all Folks ❤️

**Comment by OtacilioN** [comment]
Hey @jonathangreco I would love to see #2462 merged too, but unfortunately, I have not write access, so we need to wait until they merge it, while we don't get this release, we are maintaining a fork with this fix and you can temporally point to this fork while they don't release a new patch.

You can do this by changing your ```package.json``` import to ```    "native-base": "git+https://github.com/CESARBR/NativeBase.git#b81cc6d43d88388e112600153b4ad0b39e9beba2",```

**Comment by OtacilioN** [comment]
You can contribute with a thumbs up in this pr #2462

**Comment by alainib** [comment]
ios-funnel-outline is not available on "react-native-elements 1.1.0"

**Comment by SupriyaKalghatgi** [comment]
@alainib This is not the appropriate place to point out about react native elements
